### PR TITLE
Fixed issue #6359

### PR DESCRIPTION
**Pull request by akathuria289**
## Issue Reference
#6359

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@akathuria289 
Not working, please find the screenshot [Here](https://prnt.sc/-dOjn6UlsiZg)

**Comment by akathuria289** [comment]
@Sanjaybhattwebkul Please save the new product and then check the issue

**Comment by Sanjaybhattwebkul** [comment]
@akathuria289 
I retest this PR by creating a **new** simple product but still **getting the same issue.**

File: packages/Webkul/Admin/src/Resources/views/catalog/products/edit.blade.php [hunk 1]
```diff
@@ -115,7 +115,7 @@
                                         }
 
                                         if ($attribute->type == 'price') {
-                                            array_push($validations, 'decimal');
+                                            array_push($validations, 'decimal:2');
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Resources/views/catalog/products/edit.blade.php:118]: This is validation I guess. Check the input field.

**Review by devansh-webkul — APPROVED** [review]

**Review by Sanjaybhattwebkul — APPROVED** [review]

**Pull request merged**